### ISSUE TITLE
BUG: Error SMTP - Gmail - Send email

### ISSUE THREAD

**Issue by TCaldato** [TCaldato/iphilosophy_store#41]
When deploying on Heroku and try to register a new user, I am Getting the issue:
 - SMTP.starttls() got an unexpected keyword argument 'keyfile'

Same Issue when make a payment using Stripe: 
 - payment_intent.succeeded:
  Response
  HTTP status code
  500 (Internal Server Error)
 SMTP.starttls() got an unexpected keyword argument 'keyfile'

**Comment by TCaldato** [comment]
- Searching on slack *https://stackoverflow.com/questions/77482831/smtp-starttls-got-an-unexpected-keyword-argument-keyfile*, I follow the advice to change for Django==4.2.7. 
- Changed for Django==4.2.7 but I had to change the asgiref to asgiref==3.6.0. 
- But now, the problem was with PostgreSQL version. When elephantSQL created an instance, I did not verified the version in the beginning, which was 11.19. That caused another issue deploying to Heroku.
